src/public/services/AdminFormService.ts

```diff
@@ -29,3 +29,10 @@ export const updateSingleFormField = async (
     )
     .then(({ data }) => data)
 }
+
+export const deleteFormLogic = async (
+  formId: string,
+  logicId: string,
+): Promise<true> => {
```

Comment:
is this function typed correctly? it seems to be returning an `AxiosResponse`, not `true`. looks like we need a `.then(() => true)`.

Reply:
edited